client/src/AdminPanel/AdminPanel.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { Formik, Form } from 'formik';
+
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import Divider from '@material-ui/core/Divider';
+import Typography from '@material-ui/core/Typography';
+import Select from '@material-ui/core/Select';
+import MenuItem from '@material-ui/core/MenuItem';
+
+import { useStyles } from './adminPanel.styles';
+import CustomDialog from '../common/components/CustomDialog';
+import { useAdminPanel, useFormikCreate } from './adminPanel.hooks';
+import TextField from '../common/components/Formik/TextField';
+import Checkbox from '../common/components/Formik/Checkbox';
+import AdminTable from './AdminTable';
+
+const AdminPanel = () => {
+    const classes = useStyles();
+    const {
+        list: { total, refreshPage },
+        create: [createOn, setCreateOn],
+        usersLimit: [limit, setLimit],
+        table: { rows, deleteUser },
+    } = useAdminPanel();
+    const formikCreate = useFormikCreate(refreshPage);
+
+    return (
+        <Container className={classes.container}>
+            <Typography component="div" variant="h4" gutterBottom>
+                Users
+            </Typography>
+            <Typography paragraph gutterBottom>
+                Total of <b>{total}</b> user accounts.
+            </Typography>
+            <Grid container justify="center" spacing={1}>
+                <Grid item>
+                    <Button
+                        variant="contained"
+                        className={classes.createButton}
+                        onClick={() => setCreateOn(true)}
+                    >
+                        Create User
+                    </Button>
+                </Grid>
+                <Grid item>
+                    <Button variant="contained" onClick={refreshPage}>
+                        Refresh
+                    </Button>
+                </Grid>
+            </Grid>
+            <Grid
+                className={classes.limit}
+                container
+                justify="center"
+                alignItems="center"
+                spacing={2}
+            >
+                <Grid item>
+                    <Typography>Limit</Typography>
+                </Grid>
+                <Grid item>
+                    <Select
+                        variant="outlined"
+                        value={limit}
+                        onChange={event =>
+                            setLimit(event.target.value as number)
+                        }
+                        classes={{ select: classes.select }}
+                    >
+                        <MenuItem value={50}>50</MenuItem>
```

Comment:
This are quantified values so I believe they should be stored in some kind of `enum` with descriptive name